### ISSUE TITLE
Problem getting started with <FocusWithin>

### ISSUE THREAD

**Issue by wolfgang42** [ferdaber/react-focus-within#2]
This library is *exactly* what I need for a project. However, I can't get it to work at all for some reason. I have a very simple test script, which seems to just ignore the \<button\> focus completely:

```typescript
import * as React from 'react'
import * as ReactDOM from 'react-dom'
import FocusWithin from 'react-focus-within'

class Test extends React.Component {
	render() {
		return <FocusWithin onFocus={() => console.log('focus pocus')}>{({ isFocused }) => (
			<button className={isFocused?'bg-success':'bg-primary'}>Go</button>
		)}</FocusWithin>
	}
}

ReactDOM.render(<Test/>, document.getElementById('app'))
```

I'm using react 16.8.5 and react-focus-within 2.0.1. Help would be greatly appreciated!

**Comment by ferdaber** [comment]
In order for `FocusWithin` to detect focus events from managed children, you have to spread the provided `focusProps` to every component that you want to manage.

So given your example, it'd be like this:
```typescript
import * as React from 'react'
import * as ReactDOM from 'react-dom'
import FocusWithin from 'react-focus-within'

class Test extends React.Component {
	render() {
		return <FocusWithin onFocus={() => console.log('focus pocus')}>{({ isFocused, focusProps }) => (
			<button {...focusProps} className={isFocused?'bg-success':'bg-primary'}>Go</button>
		)}</FocusWithin>
	}
}

ReactDOM.render(<Test/>, document.getElementById('app'))
```

**Comment by wolfgang42** [comment]
Aha! I had missed that part of the documentation and assumed that `focusProps` was only for passing in properties through `withFocusWithin()`. I was confused because I was looking both the 'Props API' section and the 'Simple Usage' section and in scrolling back and forth I apparently didn't see the explanation for that argument. Would you be interested in a PR that rearranges the readme a bit to make it easier to follow for people who are getting started?

**Comment by ferdaber** [comment]
Go for it!

**Comment by ferdaber** [comment]
Fixed with #3
